code/Test_definitions/device-reachability-status-subscriptions.feature

```diff
@@ -0,0 +1,225 @@
+    When the request "createSubscription" is sent
+    Then the response code is 201
+    And if the device reachability is changed to sms usage
+    Then event notification "reachability-sms" is received on callback-url
+    And sink credentials are received as expected
+    And notification body complies with the OAS schema at "##/components/schemas/EventReachabilitySms"
+    And type="org.camaraproject.device-reachability-status-subscriptions.v0.reachability-sms"
+
+@reachability_status_subscriptions_09_Receive_notification_when_device_reachability_changed_to_disconnected
+  Scenario: Receive notification for reachability-disconnected event
+    Given that subscriptions are created synchronously
+    And a valid subscription request body  
+    And the request body property "$.type" is "reachability-disconnected"	
+    When the request "createSubscription" is sent
+    Then the response code is 201
+    And if the device reachability is changed to disconnected
+    Then event notification "reachability-disconnected" is received on callback-url
+    And sink credentials are received as expected
+    And notification body complies with the OAS schema at "##/components/schemas/EventReachabilityDisconnected"
+    And type="org.camaraproject.device-reachability-status-subscriptions.v0.reachability-disconnected"
+
+@reachability_status_subscriptions_10_subscription_expiry
+Scenario: Receive notification for subscription-ends event on expiry  
+    Given that subscriptions are created synchronously
+    And a valid subscription request body
+    And the request body property "$.subscriptionExpireTime" is set to a value in the near future
+    When the request "createSubscription" is sent
+    Then the response code is 201 
+    Then the subscription is expired
+    Then event notification "subscription-ends" is received on callback-url
+    And notification body complies with the OAS schema at "##/components/schemas/EventSubscriptionEnds"
+    And type="org.camaraproject.geofencing-subscriptions.v0.subscription-ends"
+    And the response property "$.terminationReason" is "SUBSCRIPTION_EXPIRED"
+
+@reachability_status_subscriptions_11_subscription_end_when_max_events
+   Scenario: Receive notification for subscription-ends event on max events reached 
+    Given that subscriptions are created synchronously
+    And a valid subscription request body  
+    And the request body property "$.type" is "reachability-data"
+    And the request body property "$.subscriptionMaxEvents" is set to 1 
+    When the request "createSubscription" is sent
+    Then the response code is 201 
+    Then event notification "reachability-data" is received on callback-url
+    Then event notification "subscription-ends" is received on callback-url
+    And notification body complies with the OAS schema at "##/components/schemas/EventSubscriptionEnds"
+    And type="org.camaraproject.geofencing-subscriptions.v0.subscription-ends"
```

Comment:
The type should be:
`org.camaraproject.device-reachability-status-subscriptions.v0.subscription-ends`